azure_functions_worker/logging.py

```diff
@@ -76,6 +77,11 @@ def enable_console_logging() -> None:
 
 
 def is_system_log_category(ctg: str) -> bool:
-    # Category starts with 'azure_functions_worker' or
-    # 'azure_functions_worker_errors' will be treated as system logs
-    return ctg.lower().startswith(SYSTEM_LOG_PREFIX)
+    # Category starts with the following name will be treated as system logs,
```

Comment:
@vrdmr, this is a very good point, and I don't even think we need to use the ctg.lower(). In the official [Python logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html#using-logging-in-multiple-modules), the record.name is actually case sensitive. In the code snippet they share below, the handler is not calling lower() to the record.name.

```python
def handleLogRecord(self, record):
    # if a name is specified, we use the named logger rather than the one
    # implied by the record.
    if self.server.logname is not None:
        name = self.server.logname
    else:
        name = record.name
    logger = logging.getLogger(name)
```